pkgs/by-name/co/cosmic-edit/package.nix

```diff
@@ -1,32 +1,28 @@
-{
-  lib,
-  stdenv,
-  fetchFromGitHub,
-  rustPlatform,
-  makeBinaryWrapper,
-  cosmic-icons,
-  just,
-  pkg-config,
-  libglvnd,
-  libxkbcommon,
-  libinput,
-  fontconfig,
-  freetype,
-  mesa,
-  wayland,
-  xorg,
-  vulkan-loader,
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, makeBinaryWrapper
+, mesa
+, libglvnd
+, cosmic-icons
+, just
+, pkg-config
+, glib
+, libxkbcommon
+, wayland
+, xorg
 }:
 
 rustPlatform.buildRustPackage rec {
-  pname = "cosmic-edit";
+  pname = "cosmic-files";
```

Comment:
```suggestion
```
This isn't necessary for anything, right?

Related:
- #277994

Reply:
I could remove pname if there is another thing to use, I think the other packages use it?